app/views/dashboard/index.html.erb

```diff
@@ -2,26 +2,37 @@
 
 <section>
   <%= c 'box', style: :large do %>
-    <%= c 'wrapper' do %>
+    <%= c 'wrapper', style: :narrow do %>
```

Comment:
I’d personally prefer the heading and icon list to be aligned, as they have a closer association.

<img width="1038" alt="Screen Shot 2021-06-07 at 10 44 24" src="https://user-images.githubusercontent.com/1512805/120995690-eca8df00-c785-11eb-8273-82aeff6877ba.png">